src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.cs

```diff
@@ -114,9 +119,9 @@ protected override async Task<byte[]> ReadCacheBytesAsync(string cacheKey)
                         Size = result?.Length,
                     };
 
-                    _logger.LogDebug($"[MsIdWeb] Back propagate from Distributed to Memory, cache size: {result?.Length ?? 0}. ");
+                    Log.BackPropagateL2toL1(_logger, result?.Length ?? 0, null);
                     _memoryCache.Set(cacheKey, result, memoryCacheEntryOptions);
-                    _logger.LogDebug($"[MsIdWeb] MemoryCache: Count: {_memoryCache.Count}. ");
+                    Log.MemoryCacheCount(_logger, _memoryCacheType, read, _memoryCache.Count, null);
```

Comment:
Looking at the logs, when we refresh we display a size of 0.
We could do a bit better by passing to `L2OperationWithRetryOnFailureAsync` below, line 129, the `result` after the cache key, so that the size is displayed.